lib/node_modules/@stdlib/blas/ext/base/gfill-by/README.md

```diff
@@ -174,20 +170,12 @@ gfillBy.ndarray( 3, x, 1, x.length-3, fill );
 <!-- eslint no-undef: "error" -->
 
 ```javascript
-var round = require( '@stdlib/math/base/special/round' );
-var randu = require( '@stdlib/random/base/randu' );
+var discreteUniform = require( '@stdlib/random/base/discrete-uniform' );
 var Float64Array = require( '@stdlib/array/float64' );
 var gfillBy = require( '@stdlib/blas/ext/base/gfill-by' );
 
 function fill() {
```

Comment:
Instead of this wrapper function, you could use `discreteUniform.factory( -100, 100 )`.